djorg/settings.py

```diff
@@ -21,10 +21,10 @@
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'g&amg2e#t7b&x(%an0w%ps$vfsne)-piwh3)y=!xw1l=tb3c*c'
+SECRET_KEY = config('SECRET_KEY')
```

Comment:
I'm glad you fixed this issue. Did you generate a new key since this one is still visible here?